pl.js

```diff
@@ -0,0 +1,688 @@
+settings.controller.deadZoneR Prawy koniec
+settings.controller.triggerThres Próg spustu
+settings.controller.vibration Wibracje
+settings.controller.gradSpeed Wzrastająca prędkość
+settings.controller.invert Odwróć oś Y
+settings.mobile.header Tryb mobilny
+settings.mobile.sensitivityX Czułość X
+settings.mobile.sensitivityY Czułość Y
+settings.mobile.aimsensX Czułość celowania X
+settings.mobile.aimsensY Czułość celowania Y
+settings.mobile.vibration Wibracje
+settings.mobile.gradSpeed Wzrastająca prędkość
+settings.mobile.invert Odwróć oś Y
+settings.audio.header Audio
+settings.audio.sound Uniwersalna głośność
+settings.audio.ambient Głośność otoczenia
+settings.audio.voice Głośność głosu
+settings.audio.weps Głośność broni
+settings.audio.player Głośność graczy
+settings.audio.skin Głośność elementów kosmetycznych
+settings.audio.ui Głośność interfejsu
+settings.audio.asset Głośność zasobów
+settings.audio.dialogue Głośność akcji
+settings.viewmodel.header Wyświetl model
+settings.viewmodel.fov Kąt widzenia
+settings.viewmodel.fps Kąt widzenia broni
+settings.viewmodel.bobbing Ruchliwość broni
+settings.viewmodel.leaning Oparcie broni
+settings.viewmodel.show.primary Wyświetl główną broń
+settings.viewmodel.show.secondary Wyświetl drugą broń
+settings.viewmodel.roundArms Okrągłe ręce
+settings.viewmodel.show.melee Pokaż broń białą
+settings.viewmodel.show.ads Ukryj broń przy celowaniu
+settings.viewmodel.toggleAim Przełącz celowanie
+settings.viewmodel.show.hands Wyświetl dłonie
+settings.viewmodel.leftHand Tryb leworęczny
+settings.viewmodel.aimAnim Animacja celowania
+settings.viewmodel.rot Obrócenie broni
+settings.viewmodel.xOff Przesunięcie broni w osi X
+settings.viewmodel.yOff Przesunięcie broni w osi Y
+settings.viewmodel.zOff Przesunięcie broni w osi Z
+settings.viewmodel.weapSwapY Zmiana broni Y
+settings.viewmodel.weapReloadY Przeładowanie broni Y
+settings.viewmodel.yOffADS Presunięcie Y przy celowaniu
+settings.viewmodel.adsFovMlt Siła FOV celowania
+settings.editing.header Edycja
+settings.editing.health.high Wysokie HP w HUD
+settings.editing.health.low Niskie HP w HUD
+settings.editing.speedLines Linie prędkości
+settings.editing.spdLinesCol Kolor linii prędkości
+settings.editing.score.color Kolor wyskakującego wyniku
+settings.editing.score.show Pokaż wyskakujący wynik
+settings.editing.score.scale Skala wyskakującego wyniku
+settings.editing.score.shadow Cień wyskakującego wyniku
+settings.editing.scoreOffX Przesunięcie wyskakującego wyniku w osi X
+settings.editing.scoreOffY Przesunięcie wyskakującego wyniku w osi Y
+settings.editing.prog.color Kolor paska postępu
+settings.editing.prog.opacity Przezroczystość paska postępu
+settings.editing.prog.scale Skala paska postępu
+settings.editing.prog.shadow Cień paska postępu
+settings.editing.progOffX Przesunięcie paska postępu w osi X
+settings.editing.progOffY Przesunięcie paska postępu w osi Y
+settings.editing.progRot Obrócenie paska postępu
```

Comment:
Translation of lines 329-335 from en.js (they should be put here after line 328 in this code):
settings.editing.progm.color Kolor paska ładowania
settings.editing.progm.opacity Przezroczystość paska ładowania
settings.editing.progm.scale Skala paska ładowania
settings.editing.progm.shadow Cień paska ładowania
settings.editing.progmOffX Przesunięcie baska ładowania w osi X
settings.editing.progmOffY Przesunięcie baska ładowania w osi Y
settings.editing.progmRot Obrócenie paska ładowania